pymongo/read_preferences.py

```diff
@@ -151,6 +151,14 @@ def tag_sets(self) -> _TagSets:
         set, ``{}``, means "read from any member that matches the mode,
         ignoring tags." MongoClient tries each set of tags in turn
         until it finds a set of tags with at least one matching member.
+        For example, to only send a query to an analytic node::
+
+           Nearest(tag_sets=[{"node":"analytics"}])
+
+        This can also be done using SecondaryPreferred to ensure that
+        the queries are sent to secondary nodes::
```

Comment:
I worry "to ensure that the queries are sent to secondary nodes" is a little misleading since that's not the behavior of SecondaryPreferred, How about just:
```
        For example, to only send a query to an analytic node::

           Nearest(tag_sets=[{"node":"analytics"}])

        Or using :class:`SecondaryPreferred`::

           SecondaryPreferred(tag_sets=[{"node":"analytics"}])
```